libs/upstage/langchain_upstage/layout_analysis_parsers.py

```diff
@@ -136,8 +136,8 @@ def __init__(
                                                              Defaults to "html".
             split (SplitType, optional): The type of splitting to be applied.
                                          Defaults to "none" (no splitting).
-            use_ocr (bool, optional): Extract text from images in the document.
-                                      Defaults to False. (Use text info in PDF file)
+            use_ocr (bool, optional): Extract text from images in the document with OCR.
+                                      Defaults to None, Use default behavior of the API
```

Comment:
same here. please add the link of developer doc.

Reply:
I have applied the changes you suggested, including this one.